docs/current_docs/api/arguments.mdx

```diff
@@ -310,9 +376,23 @@ For instance instead of the `String[] names` argument in the example, you can ha
 
 Here is an example call for this Dagger Function:
 
+<Tabs groupId="shell">
+<TabItem value="System shell">
+```shell
+dagger -c 'hello John,Jane'
+```
+</TabItem>
+<TabItem value="Dagger Shell">
+```shell title="Type 'dagger' to start Dagger Shell in interactive mode and enter the command below."
+hello John,Jane
```

Comment:
Actually this should work:
```
hello John Jane
```
It's the same reason it works for `withExec`. The convenience is there if there's only one required argument that is a list of strings. In that case all positional arguments get compiled into the same list argument.

However, I'm not suggesting you change it here to avoid having to teach an exception. Could be a tip, or can be left out.